Purchases/Public/CustomerInfo.swift

```diff
@@ -279,6 +281,28 @@ import Foundation
 
 }
 
+enum CustomerInfoError: Int, Error, DescribableError {
```

Comment:
doesn't describableerror already inherit from error?

Reply:
Ooops, yeah, you're right!

Reply:
✅